src/button/button.directive.ts

```diff
@@ -34,6 +34,19 @@ export class Button implements OnInit {
 	 * Specify the size of the button
 	 */
 	@Input() size: ButtonSize;
+	/**
+	 * If assistive text is used, this specifies the placement.
+	 * Possible placements are `top`, `bottom`, `left`, `right`.
+	 * If assistive text is not used, this can be left undefined.
+	 */
+	@Input() assistiveTextPlacement: "top" | "bottom" | "left" | "right";
```

Comment:
Is there a sane default we can use for this and the alignment input? Perhaps `top` `center`?

Reply:
If we have an initial value then the tooltip classes would be applied to buttons that don't have the assistive text. It wouldn't really do anything besides just being there though.

Reply:
Could we make it conditional on `this.iconOnly === true`? That way users don't have to set the placement and alignment for simple cases

Reply:
That's a good idea!